WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+class GutenbergSettings: NSObject {
```

Comment:
Continuing with the clean up, I believe this class is not longer called from an Obj-C side, so we can keep this as a pure Swift class and also we can remove the `@objc` methods.